records_mover/session.py

```diff
@@ -156,6 +163,9 @@ def __init__(self,
                  default_aws_creds_name: Union[None, str, PleaseInfer] = PleaseInfer.token,
                  default_gcp_creds_name: Union[None, str, PleaseInfer] = PleaseInfer.token,
                  session_type: Union[str, PleaseInfer] = PleaseInfer.token,
+                 # TODO: FIgure out if this should be
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker